frontend/src/app/shared/file-table/file-table.component.html

```diff
@@ -1,22 +1,54 @@
 <div class="mb-3">
-  @if (pages().totalElements) {
-    <div class="d-flex flex-row-reverse mt-3">
-      <oib-pagination [page]="pages()" (pageChanged)="currentPage.set($event)" />
+  <!-- Selection Controls - Above the table for better UX -->
+  <div class="d-flex justify-content-between align-items-center mb-3">
+    <div class="d-flex align-items-center selection-controls">
+      <button
+        type="button"
+        class="btn btn-sm"
+        (click)="selectAll()"
+        [disabled]="mainFilesCheckboxState() === 'CHECKED'"
```

Comment:
To rework a bit (maybe just renaming it ?) when the main checkbox will be removved